app/src/features/projects/components/ProjectLocationForm.tsx

```diff
@@ -32,9 +47,54 @@ export interface IProjectLocationFormProps {
  * @return {*}
  */
 const ProjectLocationForm: React.FC<IProjectLocationFormProps> = (props) => {
+  const classes = useStyles();
+
   const formikProps = useFormikContext<IProjectLocationForm>();
 
-  const { values, touched, errors, handleChange, handleSubmit } = formikProps;
+  const { values, touched, errors, handleChange, handleSubmit, setFieldValue } = formikProps;
+
+  const [bounds, setBounds] = useState<any>([]);
+  const [uploadError, setUploadError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(false);
+
+    if (!values.geometry.length || (values.geometry.length === 1 && values.geometry[0].geometry.type === 'Point')) {
```

Comment:
The length === 1 check seems pretty specific, whats the story there?

Any concern over having multiple geometries, but they are all points?

Reply:
if there is one geo only and it is a point, we cannot do the bound setting - because leaflet does not know how to handle that and tries to zoom in way too much
if there are multiple points or a polygon and a point, it is fine and knows what to do - thus, the === 1 check :)

Reply:
gotcha, maybe just add that as a comment?

Reply:
yes sir 😺 good point, will do